.github/workflows/release.yml

```diff
@@ -127,22 +127,28 @@ jobs:
 
```

Comment:
The code change looks good. It handles the case where `context.payload.pull_request` is null by checking if `pr_number` is truthy before proceeding with fetching the PR description. One suggestion would be to provide a more informative message when logging that no PR description is available, such as mentioning the specific PR number. This can help with debugging in case there are multiple PRs involved. Overall, the code change seems appropriate.